tests/Application/config/packages/setono_sylius_meilisearch.yaml

```diff
@@ -8,4 +8,4 @@ setono_sylius_meilisearch:
             entities: [ 'Setono\SyliusMeilisearchPlugin\Tests\Application\Entity\Taxon' ]
     search:
         index: products
-        hits_per_page: 2
+        hits_per_page: 30
```

Comment:
I've changed it to have more products to be sorted. It will probably be configurable from the UI in the future, I suppose 🖖